lesson_10/solid/solid_app/src/main/java/com/codedifferently/lesson10/library/MediaItem.java

```diff
@@ -0,0 +1,33 @@
+package com.codedifferently.lesson10.library;
+
+import com.codedifferently.lesson10.library.exceptions.LibraryNotSetException;
+import com.codedifferently.lesson10.library.exceptions.WrongLibraryException;
+
+public abstract class MediaItem {
+  public Library library;
+
+  abstract boolean canOrCantCheckOut();
+
+  abstract String getId();
```

Comment:
I think it would be better to return a unique ID that's not the title. Its reasonable for a library to have multiple copies of items with the same title and isbn.